executor/logging.go

```diff
@@ -10,10 +10,17 @@ import (
 )
 
 // bindLoggingPipe spawns a goroutine for passing through logging of the given output pipe.
-func bindLoggingPipe(name string, pipe io.Reader, output io.Writer, logPrefix bool) {
+//
+func bindLoggingPipe(name string, pipe io.Reader, output io.Writer, logPrefix bool, maxBufferSize int) {
```

Comment:
There don't seem to be any tests for this and the new behavior. Given that it just needs readers and writers, it should be easy enough to add a test showing exactly how this behaves with various lengths of data and configurations.

Reply:
The way I looked at it was that we don't have any tests currently, and bufio is already tested by Go's standard library, so I am not sure it would add any value, but might be complex to write and maintain. Testing was done through manual verification in this instance.